roles/upgrade/tasks/rmmod-kernel-modules.yml

```diff
@@ -3,5 +3,6 @@
   community.general.modprobe:
     name: "{{ item }}"
     state: absent
+    persistent: absent
```

Comment:
That's why remediation failed?

Reply:
Actually, `persistent: absent` doesn't work unless the module is already in a modprobe.d file.  I think the modprobe module is not sufficient for our use.  I'm working on a different way to do this.